docs/Validation.md

```diff
@@ -513,3 +513,44 @@ Now, update the xsi:schemaLocation with bad location hint
 ```
 
 In `always` you will have error, in `onValidSchema` you will have none error.
+
+## xml.validation.filters
+
+An XML validation filter gives you the capability to defines some validation rule for a given pattern file. For instance if you wish to disable validation for all files which have the `*.myxml` file extension, you must declare this filter in the `settings.json`:
```

Comment:
~~An~~ XML validation filters give you the capability to define ~~some~~ validation rules for *files matching a given pattern*.